### PR TITLE
start logging with kafka

### PR DESCRIPTION
**Pull request by karabonbl4**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+version: "3.9"
```
Comment (newDmitry) [inline docker-compose.yml]: We discussed it: all docker definitions are located in main microservice

File: src/main/java/com/kafka/StreamingApplication.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package com.kafka;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class StreamingApplication {
```
Comment (newDmitry) [inline src/main/java/com/kafka/StreamingApplication.java]: Now need to rename default spring-app-class

File: src/main/java/com/kafka/config/KafkaConfig.java [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+import org.springframework.kafka.support.serializer.JsonSerializer;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Configuration
+@EnableKafka
+public class KafkaConfig {
+    @Value("${spring.kafka.bootstrap-servers}")
+    private String bootstrapServer;
```
Comment (newDmitry) [inline src/main/java/com/kafka/config/KafkaConfig.java]: Duplicated definition of properties (another one in KafkaProperties)
Please, make the same trick with properties in config like in Library microservice

**Comment by newDmitry** [comment]
Dont use exact technology name in such common places like project name.
What if not so good technology was chosen for this functionality and it will be removed in future?

It is the same thing like: you dont use Db-name in names of services which works with DB-repository-classes.

File: docker-compose.yml (hunk 1 above)
Comment (karabonbl4) [reply docker-compose.yml]: Docker definitions of this microservice also are had in main microservice, here i tried to start it in other container and forgot to remove in commit.

File: src/main/java/com/kafka/config/KafkaConfig.java (hunk 3 above)
Comment (karabonbl4) [reply src/main/java/com/kafka/config/KafkaConfig.java]: Ok

File: src/main/java/com/kafka/StreamingApplication.java (hunk 2 above)
Comment (karabonbl4) [reply src/main/java/com/kafka/StreamingApplication.java]: Maybe i try to rename project name? Then it will be right

**Pull request merged**